pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
@rsandell I don't know about the CI env but I consider it is very helpful for local development environment.

+1 for enforcing FindBugs.